### ISSUE TITLE
[Admin] Current page referencing to the object id instead of the current page.

### ISSUE THREAD

**Issue by istinspring** [viewflow/django-material#154]
```
[1] - http://localhost:8000/admin/locations/location/1
[2] - http://localhost:8000/admin/locations/location/?p=1
```

Django 1.9

**Comment by kmmbvnr** [comment]
i don't get what you mean

**Comment by istinspring** [comment]
I got it.

https://github.com/viewflow/django-material/blob/master/material/admin/templatetags/material_admin.py#L181

```
        return format_html('<li class="active"><a href="{0}">{0}</a></li> ',
                           i+1,
                           cl.get_query_string({PAGE_VAR: i}))
```

So for current page it will set {0} to ```the current_page_number+1```  so it would be relative url to the object with id "current_page_number+1".

You need to change:

```
        return format_html('<li class="active"><a href="/?p={0}">{0}</a></li> ',
                           i+1,
                           cl.get_query_string({PAGE_VAR: i}))
```

to the ```/?p={0}```, actually it's not as easy there could be other parameters in url.

**Comment by istinspring** [comment]
can try to do pull request if you want

**Comment by kmmbvnr** [comment]
Yep, PRs are welcome.

You can take a look how this functionality implemented in django admin template tags

**Comment by istinspring** [comment]
ok

**Comment by istinspring** [comment]
done

**Comment by kmmbvnr** [comment]
fixed in 9bf19db06f15733fcdb6f3ffc6b0037fce5bb60e
